rb/build.desc

```diff
@@ -117,7 +117,8 @@ ruby_library(name = "firefox",
     "lib/selenium/webdriver/firefox.rb"
   ],
   resources = [
-    { "//javascript/webdriver/atoms:getAttribute": "rb/lib/selenium/webdriver/atoms/getAttribute.js"}
+    { "//javascript/webdriver/atoms:getAttribute": "rb/lib/selenium/webdriver/atoms/getAttribute.js"},
+    { "//javascript/firefox-driver:webdriver_prefs" : "rb/lib/selenium/webdriver/firefox/extension/prefs.json" }
```

Comment:
I know python still use [this here](https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/firefox/firefox_profile.py#L62-L65), and it is mentioned in other build.desc files (didn't check to see where it was used).  The source file itself [is here](https://github.com/SeleniumHQ/selenium/blob/master/javascript/firefox-driver/webdriver.json). I'm not sure what all default preferences are desired, but there's a lot in there.